apps/aehttp/src/swagger/swagger_server.erl

```diff
@@ -1,7 +1,7 @@
 -module(swagger_server).
 
 
--define(DEFAULT_ACCEPTORS_POOLSIZE, 100).
+-define(DEFAULT_ACCEPTORS_POOLSIZE, 10).
```

Comment:
I see that this change gets lost after regenerating Swagger, though this seems inevitable at the moment as it is [hardcoded in upstream codegen](https://github.com/swagger-api/swagger-codegen/blob/28467a5c3f86eac33aa5391a024f4fbf62364646/modules/swagger-codegen/src/main/resources/erlang-server/server.mustache#L4). Ok.

Reply:
Yes. I personally think that 100 is a ridiculously large acceptor pool, since it's always replenished, and therefore only needs to be large enough not to run out before the pool manager can start new acceptors.